src/profile/v100/address/legacy_olympia_account_address_uniffi_fn.rs

```diff
@@ -0,0 +1,118 @@
+use crate::prelude::*;
+
+#[uniffi::export]
+pub fn new_legacy_olympia_account_address_sample() -> LegacyOlympiaAccountAddress
+{
+    LegacyOlympiaAccountAddress::sample()
+}
+
+#[uniffi::export]
+pub fn new_legacy_olympia_account_address_sample_other(
+) -> LegacyOlympiaAccountAddress {
+    LegacyOlympiaAccountAddress::sample_other()
+}
+
+#[uniffi::export]
+pub fn new_legacy_olympia_account_address_from_public_key(
+    public_key: Secp256k1PublicKey,
+) -> LegacyOlympiaAccountAddress {
+    LegacyOlympiaAccountAddress::from(public_key)
+}
+
+#[uniffi::export]
+pub fn new_legacy_olympia_account_address_from_string(
+    string: String,
+) -> Result<LegacyOlympiaAccountAddress> {
+    LegacyOlympiaAccountAddress::from_str(&string)
+}
+
+#[uniffi::export]
+pub fn legacy_olympia_account_address_to_string(
+    address: &LegacyOlympiaAccountAddress,
+) -> String {
+    address.to_string()
+}
+
+#[uniffi::export]
+pub fn legacy_olympia_account_address_to_babylon_account_address(
+    address: LegacyOlympiaAccountAddress,
+) -> AccountAddress {
+    address.to_babylon_account_address()
+}
+
+#[uniffi::export]
+pub fn legacy_olympia_account_address_is_legacy_of_babylon(
```

Comment:
We can use this in wallets part of import Olympia wallet flow:
https://github.com/radixdlt/babylon-wallet-ios/blob/main/RadixWallet/Clients/ImportLegacyWalletClient/ImportLegacyWalletClient%2BLive.swift#L134-L158

and analoguosly in android.